### DISCUSSION TITLE
NADA BUILD

### DISCUSSION THREAD

**Discussion by ScapeXYZ** [NillionNetwork/discussions#135] [Bugs]
NADA build
`[[programs]]`
`path = "src/secret_addition.py`
`name = "secret_addition`
`prime_size = 128`
`nada-project.toml file`

This is the error I'm getting 
Error I'm getting
nada build
Error:
0: TOML parse error at line 1, column 1
0: |
0: 1 | [[programs]]
0: | ^
0: missing field name
0:

Location:
tools/nada/src/nada_project_toml.rs:67

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.

**Comment by crypblizz8** [comment]
This was solved? @ScapeXYZ

**Reply by ScapeXYZ** [reply]
Yes
But I've cancelled it and start from the beginning I missed one step then
